### ISSUE TITLE
Script Execution Failure at nala 94% on Debian-Titus BSPWM Branch

### ISSUE THREAD

**Issue by TechsTune** [ChrisTitusTech/Debian-titus#106]
I have an issue with Debian-Titus (bspwm branch). During the execution of the script, the process halted at 94% completion with Nala, and my screen suddenly lost signal. so, I restarted my computer with the power button and ran the script again, it then rebooted and it briefly displayed
“[FAILED] Failed to start polkit.service - Authorization 
Manager” before returning me to the terminal interface again.

**Comment by blusewill** [comment]
Maybe try manually install it? Since that script is kinda outdated?

**Comment by TechsTune** [comment]
Sorry for my late response. I tried to manually install `policykit-1` before executing `./install.sh`, but that unfortunately did not resolve the issue. My monitor lost signal at the beginning of the script, so I had to reboot my PC again. After executing the script again, I encountered this error:

```
Error: E: Unable to correct problems, you have held broken packages.
Synchronizing state of lightdm service with SysV service script with /usr/lib/systemd/systemd-sysv-install.
Executing: /usr/lib/systemd/systemd-sysv-install enable lightdm
The unit files have no installation config (WantedBy=, RequiredBy=, UpheldBy=,
Also=, or Alias= settings in the [Install] section, and Default Instance= for
template units). This means they are not meant to be enabled or disabled using systemctl.
```

I appreciate your help.

**Comment by blusewill** [comment]
You should install some polkit manager service such as gnome-keyring I think.
That should resolved the issue

**Comment by TechsTune** [comment]
I tried to manually install `lxpolkit` and `gnome-keyring` before running the script but the error persists. When I executed `sudo lightdm`, I got the login screen, but after entering my credentials, it didn’t bring me to the desktop. After logging In a few times, the system froze.

**Comment by crazycheetah42** [comment]
Is this on a system you already have data on? Because you might have to reinstall Debian and run the script from a fresh install.
Or you can remove polkit and then try running the script to see if it installs?

**Comment by TechsTune** [comment]
No, this is happening on a fresh system. I managed to resolve the issue where Nala couldn’t install packages.

```
# Essential Programs
essential_programs=(feh bspwm sxhkd kitty arandr rofi polybar picom thunar gvfs lxpolkit x11-xserver-utils unzip yad wget pulseaudio pavucontrol gnome-keyring accountsservice lightdm neovim)
# Other less important Programs
other_programs=(lxappearance papirus-icon-theme fastfetch flameshot psmisc fonts-noto-color-emoji curl ntfs-3g zoxide)
# Install programs individually
for program in "${essential_programs[@]}" "${other_programs[@]}"; do
  nala install -y "$program" || { echo "Failed to install $program" >&2; exit 1; }
  sleep 1
done
```

However, I'm now facing an issue where my system radonmly crashes during the installation. This crash also seems to occur when launching other graphical applications from the TTY, for example, with `xinit`. After logging in, the issue no longer seems to persist.

**Comment by crazycheetah42** [comment]
That's weird but where did you get the code you posted? Doesn't seem to be in the install script.

**Comment by TechsTune** [comment]
After i got an error from Nala, I modified the installation script to loop through and install each package individually, which resolved that issue. Unfortunately, My system still crashes. I checked the logs using both dmesg and journalctl, and I also installed kdump, but none of them provided any useful information.
